### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by nicatino**

### CODE REVIEW

File: src/validateEmail.test.js [hunk 1]
```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
...
+    expect(validateEmail('')).toEqual(expect.any(Boolean));
+  });
+
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail('adolas666@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the email with integers`, () => {
+    expect(validateEmail('adolas666@gmail.com')).toBe(true);
+  });
```
Comment (murat-yesimbietov) [inline src/validateEmail.test.js:20]: What is the difference between these tests?

File: src/validateEmail.test.js [hunk 2]
```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
...
+
+  it(`should return 'true' for the email '_'`, () => {
```
Comment (murat-yesimbietov) [inline src/validateEmail.test.js:22]:
```suggestion
  it(`should return 'true' for the email with '_' in 'personal_info' part`, () => {
```

File: src/validateEmail.test.js [hunk 3]
```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
...
+    if email doesnt start or ends or doeasnt has two dots in row`, () => {
+    expect(validateEmail('adolas.666@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the email which domain has:
+    -, . and integers`, () => {
+    expect(validateEmail('adolas.666@gma1l.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the email which domain has - in it `, () => {
```
Comment (murat-yesimbietov) [inline src/validateEmail.test.js:40]:
```suggestion
  it(`should return 'true' for the email which domain has '-' in it `, () => {
```

File: src/validateEmail.test.js [hunk 4]
```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
...
+
+  it(`should return 'false' for the email that starts with '.'`, () => {
+    expect(validateEmail('.adolas666@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email that ends with '.'`, () => {
+    expect(validateEmail('adolas666.@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email that has two '.' in row`, () => {
```
Comment (murat-yesimbietov) [inline src/validateEmail.test.js:61]: add more context: two in row dots in the 'personal_info' part

File: src/validateEmail.test.js [hunk 5]
```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
...
+  // Negaive scenario:
+  it(`should return 'false' for the email with non ASCII characters`, () => {
+    expect(validateEmail('adolasсус@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email that starts with '.'`, () => {
+    expect(validateEmail('.adolas666@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email that ends with '.'`, () => {
```
Comment (murat-yesimbietov) [inline src/validateEmail.test.js:57]: Add more context: dot at the end of the personal_info part

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]

File: src/validateEmail.test.js (hunk 1 above)
Comment (nicatino) [reply src/validateEmail.test.js]: Repaired

**Review by murat-yesimbietov — APPROVED** [review]

**Pull request closed without merging**